src/keystore.h

```diff
@@ -17,15 +17,11 @@
 /** A virtual base class for key stores */
 class CKeyStore
 {
-protected:
-    mutable CCriticalSection cs_KeyStore;
-
 public:
     virtual ~CKeyStore() {}
```

Comment:
Destructor should also be pure?

Reply:
@Empact see https://stackoverflow.com/a/3628611.